test-e2e/selectors.js

```diff
@@ -0,0 +1,54 @@
+// Onboarding
+export const onboarding_title = '[data-e2e=onboarding_title]'
```

Comment:
btw, we'll discuss more, but i think this is an antipattern. we should directly inline the selector when we need them.
and we should also help you on more relevant selector. like to me we can afford to use id and classes so just do normal `#foo .bar`